src/cli/commands/mod.rs

```diff
@@ -0,0 +1,98 @@
+pub(crate) mod app;
+pub(crate) mod auth;
+pub(crate) mod certificate;
+pub(crate) mod channel;
+pub(crate) mod environment_variable;
+pub(crate) mod new;
+pub(crate) mod revision;
+
+use clap::{AppSettings, Subcommand};
+use std::path::PathBuf;
+
+const INVOICE_VERSION_ACCEPTED_VALUES: &[&str] = &["dev", "development", "prod", "production"];
+
+#[derive(Subcommand)]
+pub(crate) enum Commands {
+    /// Add, update, and remove Applications
+    #[clap(subcommand)]
+    App(app::Commands),
```

Comment:
nit: would it be more readable if command enums were directly named like `AppCommands` or `AuthCommands`?

Reply:
I figured the namespace would be enough of an indicator… Is the convention in Rust to “stutter” and re-repeat the namespace in the type name (`app::AppCommands`)? I thought most language linters discourage that convention

Reply:
You would `use app::AppCommands` in the top level, and throughout the file you'd use `AppCommands` directly.

Reply:
Clippy has a linting rule against types that stutter (`clippy::module_name_repetitions`), but it is disabled by default. Names like `app::AppCommands` is generally discouraged.